mapper/pojo/src/main/java/org/hibernate/search/mapper/pojo/bridge/mapping/impl/IdentifierMappingImplementor.java

```diff
@@ -4,18 +4,19 @@
  * License: GNU Lesser General Public License (LGPL), version 2.1 or later
  * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
  */
-package org.hibernate.search.mapper.pojo.bridge.impl;
+package org.hibernate.search.mapper.pojo.bridge.mapping.impl;
 
 import java.util.function.Supplier;
 
+import org.hibernate.search.mapper.pojo.bridge.mapping.spi.IdentifierMapping;
 import org.hibernate.search.mapper.pojo.mapping.context.spi.AbstractPojoMappingContextImplementor;
 import org.hibernate.search.mapper.pojo.session.context.spi.AbstractPojoSessionContextImplementor;
 
 /**
  * @param <I> The identifier type for the mapped entity type.
  * @param <E> The entity type mapped to an index.
  */
-public interface IdentifierMapping<I, E> extends AutoCloseable {
+public interface IdentifierMappingImplementor<I, E> extends IdentifierMapping, AutoCloseable {
```

Comment:
Extending `IdentifierMapping', I think we can remove `toDocumentIdentifier` method from here.